src/lib/util/Config.ts

```diff
@@ -218,6 +229,24 @@ export class Config {
 		return this.getConfigValue<string[]>(propertyName, engine, stringArrayTypeGuard, 'InvalidStringArrayValue');
 	}
 
+	protected getStringConfigValue(propertyName: string, engine: ENGINE): Promise<string> {
+		return this.getConfigValue<string>(propertyName, engine, stringTypeGuard, 'InvalidStringValue');
+	}
+
+	protected getStringEnumConfigValue(propertyName: string, engine: ENGINE, enumOptions: string[]): Promise<string> {
+		return this.getStringConfigValue(propertyName, engine)
+			.then(enumValue => {
+				if (enumOptions.includes(enumValue)) {
```

Comment:
Should this perform some additional toUpperCase() and whitespace trim before comparison?

Reply:
Will use `toLowerCase()` since the enum values are lowercase, but good catch.